packer/scripts/versions_to_json.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
```

Comment:
How much of this would actually already be covered by https://github.com/theforeman/forklift/blob/master/bin/ansible-vars?

Reply:
I could get most of the way there with just something like this:   ./bin/ansible-vars  | jq '. "centos7-katello-3.18"'

But it wouldn't get me the branch parameters to check everything out to the correct branches automatically.  I'm open to other suggestions.  Was trying to balance easy of use with complexity

Reply:
What I meant was that I would like to reuse as much logic as possible. Note it's already Ruby. You can either copy it, modify it or add additional data to versions.yaml to add this info.